src/js/flows/ingestFiles.test.ts

```diff
@@ -33,10 +32,6 @@ beforeEach(() => {
   store.dispatchAll([Clusters.add(conn), Current.setConnectionId(conn.id)])
 })
 
-afterEach(() => {
```

Comment:
was this just not actually necessary?

Reply:
I couldn't find a reason for it. I think it was old.